tools/dm-worker-intro.md

```diff
@@ -0,0 +1,81 @@
+---
+title: DM-worker Introduction
+summary: Learn the features of DM-worker.
+category: tools
+---
+
+# DM-worker Introduction
+
+DM-worker is a tool used to synchronize the data from MySQL/MariaDB in the upstream to TiDB in the downstream. 
+
+It has the following features:
+
+- Connecting to a MySQL/MariaDB instance and registering as the slave of this instance
+- Reading the binlog files of MySQL/MariaDB and persisting the binlog files to the local storage (relay logs)
```

Comment:
`Reading the binlog events from MySQL/MariaDB and persisting them to the local storage`